src/Jackett.Common/Indexers/BJShare.cs

```diff
@@ -127,13 +127,13 @@ private string StripSearchString(string term, bool isAnime)
         {
             // Search does not support searching with episode numbers so strip it if we have one
             // Ww AND filter the result later to archive the proper result
-            term = Regex.Replace(term, @"[S|E]\d\d", string.Empty).Trim();
+            term = Regex.Replace(term, @"[S|E]\d\d", string.Empty, RegexOptions.IgnoreCase).Trim();
             return isAnime ? term.TrimEnd(_digits) : term;
```

Comment:
And this one too, so we can remove of the _digits variable on Line 26.

```suggestion
            return isAnime ? Regex.Replace(term, @"\d*$", string.Empty) : term;
```